src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -105,14 +105,10 @@ private static string Concat(string container, string name)
                 }
             }
             """;
-        internal const string s_collectionExtensionsWithSpan = s_collectionExtensions +
+        internal const string s_collectionExtensionsWithReadOnlySpan = s_collectionExtensions +
```

Comment:
>s_collectionExtensionsWithReadOnlySpan [](http://example.com/codeflow?start=30&length=38)

Alternatively, could we leave the name unchanged, and simply remove the `Report<T>(this in Span<T>)` overload? Would that result in many additional test changes?

Reply:
Not many additional changes, I can do that.